src/answers-umd.js

```diff
@@ -412,14 +413,29 @@ class Answers {
    * Updates the css styles with new current variables. This is useful when the css
    * variables are updated dynamically (e.g. through js) or if the css variables are
    * added after the ANSWERS.init
+   *
+   * To solve issues with non-zero max-age cache controls for link/script assets in IE11,
+   * we add a cache busting parameter so that XMLHttpRequests succeed.
+   *
    * @param {Object} config Additional config to pass to the ponyfill
    */
   ponyfillCssVariables (config = {}) {
     cssVars({
       onlyLegacy: true,
       onError: config.onError || function () {},
       onSuccess: config.onSuccess || function () {},
-      onFinally: config.onFinally || function () {}
+      onFinally: config.onFinally || function () {},
+      onBeforeSend: (xhr, node, url) => {
+        try {
+          const uriWithCacheBust = new URL(url);
+          const params = new SearchParams(uriWithCacheBust.search);
+          params.set('_', new Date().getTime());
+          uriWithCacheBust.search = params.toString();
+          xhr.open('GET', uriWithCacheBust.toString());
+        } catch (e) {
+          // Catch the error and continue if the URL provided in the asset is not a valid URL
```

Comment:
what kinds of errors might we get here, and is it okay to just ignore them? also love that we don't import any new dependencies

Reply:
Ideally, this cache busting will not be blocking JS on the page if a query param cannot be added, thus the continue.

This affects two URLs of note: relative URLs (e.g. href="bundle.css") and relative protocol URLs (e.g. href="//www.yext.com/some.css").

If we try to use those kinds of URLs as parameters for the URL constructor, we get a JS error, not fitting the schema.